sample_data/bahamas_by_region_since_1973/bahamas_population_2010.csv

```diff
@@ -0,0 +1,33 @@
+shapeName,Cartogram Data (eg. Population),Color,Inset,Label,,,,,,,,
+Acklins,565,,,,,,,,,,,
```

Comment:
It appears to be considered as part of the Hope Town district.

Reply:
I believe the issue is that the source does not acknowledge New Providence as a district because it is not listed as an official 2nd or 3rd-tier district under the 2010 census.